api/src/main/java/uk/gov/dhsc/htbhf/claimant/scheduler/MessageProcessorScheduler.java

```diff
@@ -97,4 +97,14 @@ public void processPaymentMessages() {
     public void processReportClaimMessages() {
         messageProcessor.processMessagesOfType(MessageType.REPORT_CLAIM);
     }
+
+    @Scheduled(cron = EVERY_30_SECONDS)
+    @SchedulerLock(
+            name = "Process REPORT_PAYMENT messages",
+            lockAtLeastForString = FIFTEEN_SECONDS,
+            lockAtMostForString = ONE_HOUR)
```

Comment:
Maybe not for this PR, but it would probably be a good idea to have our own annotation which defaults the values for `lockAtLeastForString` and `lockAtMostForString` to these values because the majority of the time we want them all to be the same. If we did this, it would also mean that any that validly need to differ stick out clearly from the rest.